### DISCUSSION TITLE
防堵遠距考試作弊！清大研發「AI 鷹眼」數位監考系統

### DISCUSSION THREAD

**Discussion by Jay-722** [1101-GS4719-C/Python-Community-GS4719-C#527] [General]
為了防堵遠距線上考試的作弊問題，清華大學教務處教學發展中心，今日發表自行研發的「AI 鷹眼」數位監考系統，只要使用一般電腦的視訊鏡頭，就可以達到一對多的大班遠距監考，一旦考生的視線飄到螢幕外，或是開啟其它網頁來找答案，系統就會立即示警，提醒考生及監考者注意。

拜託不要哈哈哈，但其實這是好東西，疫情影響下考試就不怕作弊

全文:https://technews.tw/2022/03/28/ai-hawkeye/

**Comment by 11timyyy** [comment]
可以大幅降低線上作弊的發生頻率，讚讚

**Reply by Jay-722** [reply]
不錯吧

**Comment by leon10456** [comment]
到時候抬頭思考一下就被當作弊了QQ

**Reply by Jay-722** [reply]
我也會怕

**Comment by blackline0911** [comment]
我認為這能防君子不防小人。因為真的想作弊的人要躲過這些方法太簡單了，畢竟鏡頭不像人眼，無法完全監控學生，所以想作弊的還是會作弊。

**Reply by Jay-722** [reply]
不作弊的也不會作弊

**Comment by HuanYuuuuuu** [comment]
這樣好可怕 我如果只是看遠方思考一下題目也被判作弊怎麼辦

**Reply by Jay-722** [reply]
你在看遠山，逼逼

**Comment by liangyinglly** [comment]
雖然是個防止作弊的好方法，但感覺會有很多爭議，因為他這樣限制學生全程只能緊盯著螢幕

**Reply by Jay-722** [reply]
恩恩，還要設計更精確

**Comment by xkllkx** [comment]
萬一要低頭寫字或計算不就被抓了?治標不治本

**Comment by Lynn12378** [comment]
設計理念感覺是好的，但是AI在某些程度來說不如人類靈活，若只考量考生的視線飄移至螢幕外，感覺條件還不夠嚴謹

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，考試要公平才好。

**Reply by Jay-722** [reply]
當然

**Comment by d26488399** [comment]
雖然這項技術的出發點是好的，但如過容易誤判，這項技術反而容易增加考生的負擔。

**Comment by SamTU0222** [comment]
謝謝分享，這項技術還是要透過測試來觀察是否能維持遠距考試的公平性

**Comment by ghost** [comment]
這壓力真的有夠大，最怕的是沒作弊被當作有作弊

**Comment by Sazmafuji** [comment]
會不會以後考試因為誤會，雖然沒作弊變作弊

**Comment by lx3211** [comment]
感覺考試完想的都不是考試題目
而是自己有沒有被當作弊了xd

**Comment by sean1325418** [comment]
這樣準確度要很高欸，不然被機器誤會也不知道要跟誰解釋

**Comment by PoooYen** [comment]
為了公平性希望這項技術能發展順利，也可以讓國考能夠有上機考的選項

**Comment by yi-1111** [comment]
真是道高一尺魔高一丈阿

**Comment by 110409538** [comment]
線上考試有很多漏洞,很難保證公平性,有了這個監考系統,便能降低作弊的機率.

**Comment by bloodtrail0511** [comment]
侷限性有點大，感覺思考或計算的時候抬頭低頭就會被警告

**Comment by HuanYuuuuuu** [comment]
要降低作弊案例還是好好教育學生的心態吧

**Comment by cordydeng** [comment]
感覺這是一個很好的方法，不僅可以幫助到老師，也能讓學生考試時不需要擔心可能因為一個小動作而被懷疑是作弊

**Comment by Lai27330** [comment]
即使可以偵測視線轉移，但鏡頭的角度仍然有一定的限制，而且個人的作答習慣不同也容易產生爭議，像是需要手寫計算的地方就不太適用，因此感覺還是有侷限性。

**Comment by etin2468** [comment]
考試這種東西我覺得最重要的還是要對得起自己的良心啦，多了這種科技輔助可能反而害到其他的人

**Comment by henrywen547** [comment]
光抓誰誰誰作弊感覺是不夠，應該師生間要討論課綱難易度並作調整，畢竟作弊好像也只解決了被當的問題。

**Comment by Rosancl** [comment]
出發點是好的，遠距考試的情況下有監考系統還滿不錯的，但若是針對視線飄走就算作弊，實行後大概會有很多爭議情況。

**Comment by yucheng1028** [comment]
我還是覺得 自由心證比較重要啦

**Comment by sys511613** [comment]
我都把重點都記在腦袋裡，考試的時候再偷偷回想，助教根本不會發現

**Comment by treeann** [comment]
AI真的太多用途了

**Comment by jcpan821** [comment]
這樣不就沒辦法計算了

**Comment by Ben6uo** [comment]
到時候如果想放空一下腦子就直接BBQ了

**Comment by stan890425** [comment]
還敢亂喵阿~~

**Comment by alexthebigpp** [comment]
感謝分享，感覺眼睛飄出視窗外就示警有點多餘。
